### ISSUE TITLE
Appreciation for the Repo

### ISSUE THREAD

**Issue by Zeik0s** [heisath/wdmc2-kernel#3]
Hi, i also started a Repo to improve wdmc documentation on github and to document the procedure for kernel compiling. But i never made it to a working 5.x Version, my compiled kernels only worked until 4.19.
But you did it, and i will for sure upgrade mine to keep it recent. Thanks a lot.

**Comment by Zeik0s** [comment]
So i will make a hint to redirect Users to your Repo, because the quality here of the overall documentation is a lot better.

**Comment by Heisath** [comment]
Thanks for the nice words, you're welcome to redirect users here and/or send improvements via PR. 

Regarding 5.x: It seems to be neccessary to keep the kernel image below ~3.5MiB. Otherwise it does not load. I had to limit the amount of stuff compiled in the kernel and switch as much as possible off/to module to keep on going. 

Also thanks to @puxxi2003 I figured out it is better/needed to use another GCC after 5.6
`#makehelp='make CROSS_COMPILE=/opt/arm-marvell-linux-gnueabi/bin/arm-marvell-linux-gnueabi- ARCH=arm'    FOR KERNEL VERSION <= 5.6`

`makehelp='make CROSS_COMPILE=/opt/gcc-arm-none-eabi/bin/arm-none-eabi- ARCH=arm'                        #FOR KERNEL VERSION >= 5.6`
